datanode/gateway/graphql/schema.graphql

```diff
@@ -4873,6 +4872,31 @@ type ProposalTerms {
   change: ProposalChange!
 }
 
+"Terms for a batch governance proposal on Vega"
+type BatchProposalTermsChange {
+  """
+  RFC3339Nano time and date when this proposal is executed (if passed). Note that it has to be after closing date time.
```

Comment:
```suggestion
  RFC3339Nano time and date when this proposal will be executed, if it passes.
```
I don't think the note is relevant here as this is querying for an existing proposal so there's not much the user can do about it at this stage.